### DISCUSSION TITLE
Issue with ConsumerWidget: "Classes can only extends another classes"

### DISCUSSION THREAD

**Discussion by pedropacheco92** [rrousselGit/riverpod#998] [Q&A]
**Describe the bug**
Error on extending ConsumerWidget

**To Reproduce**
Just created a new flutter project with riverpod 1.0.2 and cannot use the simple counter example.
Running flutter 2.5.3

**Expected behavior**
A class to be able to extend ConsumerWidget

**Comment by rrousselGit** [comment]
You most likely did not import flutter_riverpod and/or need to run "flutter pub get"

For any other reason, I'd need a reproduction example. Otherwise I can't help

**Reply by pedropacheco92** [reply]
Ok, that was it. Thanks for the quick response :)

But this is not clear in the documentation. Other libraries, like redux, have this described in the README file, I would recommend adding this.
